samples/sampler/stories/default/Item.js

```diff
@@ -4,35 +4,29 @@ import UiItem, {ItemBase as UiItemBase} from '@enact/ui/Item';
 import React from 'react';
 import {storiesOf} from '@storybook/react';
 
-import Icon, {icons} from '@enact/agate/Icon';
+import Icon from '@enact/agate/Icon';
 import Item, {ItemBase} from '@enact/agate/Item';
 
 const Config = mergeComponentMetadata('Item', UiItemBase, UiItem, ItemBase, Item);
 Item.displayName = 'Item';
 
-const iconList = [null, ...Object.keys(icons)];
-
 storiesOf('Agate', module)
 	.add(
 		'Item',
 		() => {
-			const slotBeforeIcon = select('slotBefore Icon', iconList, Config);
-			const slotAfterIcon = select('slotAfter Icon', iconList, Config);
 			return (
 				<Item
+					centered={boolean('centered', Config)}
```

Comment:
![image](https://user-images.githubusercontent.com/34676178/104412093-f9e2e880-55ae-11eb-9abd-68f0e483ad5d.png)

Center prop's behavior looks unstable.
I don't know if there were any other requirements for `center`.
If you don't have a GUI requirement or other necessity, would you leave it remove the `center` prop?
Or do you want to solve the bug individually?